lib/src/resultsetTable/ResultsetTable.tsx

```diff
@@ -197,9 +198,10 @@ const HeaderContainer = styled.div`
       : props.theme.headerTextAlign === "right"
       ? "flex-end"
       : "flex-start"};
-  width: 100%;
+  width: fit-content;
   :focus {
-    outline: #0095ff auto 1px;
+    outline: ${(props) => (props.isSortable ? "#0095ff solid 2px" : "")};
```

Comment:
I think this is better: `${(props) => (props.isSortable && "outline: #0095ff solid 2px;")}`